go/items.json

```diff
@@ -0,0 +1 @@
+{"items":[{"name":"jacket","category":"fashion"},{"name":"cap","category":"fashion"},{"name":"denim","category":"fashion"},{"name":"sneaker","category":"fashion"}]}
```

Comment:
`git` is very hard to use isn't it?

Adding each file with `git add` is quite annoying. You can try learning about `.gitignore` which will automatically ignore adding extra files like this one. 💪

Reply:
dc7a255474d1135fa01b3688c91a29b5d981c165

Reply:
It seems that items.json file isn't ignored even though I added gitignore file and write "*.json". Could you tell me what is the problem??